api/v1alpha1/selfnoderemediationconfig_webhook.go

```diff
@@ -149,3 +154,41 @@ func (f *field) validate() error {
 
 	return nil
 }
+
+func (r *SelfNodeRemediationConfig) validateCustomTolerations() error {
+	customTolerations := r.Spec.CustomDsTolerations
+	if len(customTolerations) == 0 {
```

Comment:
redundant, the range assignment in the loop can deal with nil and empty slices

Reply:
cool, I didn't know range is valid over nil :+1: